src/defaults.js

```diff
@@ -942,6 +940,55 @@ var defaults = {
      */
     editOnNextCell: false,
 
+
+    /* COLUMN SORTING */
+
+    /**
+     * Ignore sort handling in feature/ColumnSorting.js.
+     * Useful for excluding some columns but not other from participating in sorting.
+     *
+     * @default
+     * @type {boolean}
+     * @memberOf module:defaults
+     */
+    unsortable: false,
+
+    /**
+     * Sort column on double-click rather than single-click.
+     *
+     * Used by:
+     * * feature/ColumnSorting.js to decide which event to respond to (if any, see `unsortabe`).
+     * * feature/ColumnSelection.js to decide whether or not to wait for double-click.
+     * @default
+     * @type {boolean}
+     * @memberOf module:defaults
+     */
+    sortOnDoubleClick: true,
+
+    /**
+     * **This is a standard property definition for sort plug-in use.
+     * It is not referenced in core.**
+     *
+     * The maximum number of columns that may participate in a multi-column sort (via ctrl-click headers).
+     * @default
+     * @type {number}
+     * @memberOf module:defaults
+     */
+    maxSortColumns : 3,
```

Comment:
Is there a reason to have a limit?

Reply:
@nmichaud 
> Is there a reason to have a limit?

I think the reason was the extra crunching involved. Implementations are free to ignore the property. Applications are free to set it very high. We could set the default higher (like 5 or 6)? I wouldn't eliminate it entirely; t's a handy property to keep around.

What do you think?

Reply:
Well if implementations are free to ignore it, i'm not sure how useful it is.

Reply:
We have no way of enforcing that applications respect the property values. If they do, it becomes "easier" (not _guaranteed_) that one sorting plug-in can be swapped out for another. I think it is better to have it than not to have it. And on a practical level, apps can override it if it is a problem. We can also, as I said make the default higher. What do you think?